### PR TITLE
Deprecate GET phys_locations/:id

### PR DESCRIPTION
**Pull request by mhoppa**
## What does this PR (Pull Request) do?

- [x] This PR is not related to any Issue 

## Which Traffic Control components are affected by this PR?

- Documentation
- Traffic Ops

## What is the best way to verify this PR?

Run traffic ops and make sure you can not hit the endpoint in API 2.0 and when performing a GET on it in 1.x you receive a deprecation message back in the returned alerts. 

## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR ensures that database migration sequence is correct OR this PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: docs/source/api/v1/phys_locations_id.rst [hunk 1]
```diff
@@ -114,6 +117,12 @@ Response Structure
 			"state": "DC",
 			"zip": "20500"
 		}
+	],
```
Comment (shamrickus) [inline docs/source/api/v1/phys_locations_id.rst]: The Content-Length header needs to be updated.

Comment (mhoppa) [reply docs/source/api/v1/phys_locations_id.rst]: I can update it - but keep in mind I havent done this with any of my rewrites or deprecations (unsure if many ppl have) - I dont think it is a huge deal as its just an example response structure

Comment (shamrickus) [reply docs/source/api/v1/phys_locations_id.rst]: I asked around (aka docs overlord @ocket8888) and you're correct, I wouldn't bother updating.

Comment (mhoppa) [reply docs/source/api/v1/phys_locations_id.rst]: docs overlord thats hilarious! 🤣

**Pull request merged**